src/Features/CSharp/Portable/Diagnostics/Analyzers/CSharpTypeStyleDiagnosticAnalyzerBase.cs

```diff
@@ -56,7 +57,7 @@ private void HandleVariableDeclaration(SyntaxNodeAnalysisContext context)
 
             var semanticModel = context.SemanticModel;
             var declaredType = Helper.FindAnalyzableType(declarationStatement, semanticModel, cancellationToken);
-            if (declaredType == null)
+            if (declaredType == null || declaredType.IsKind(SyntaxKind.NullableType))
```

Comment:
i'm wary about this.   my understanding is that it will soon be fine to use `var` with nullable types.  @jcouv  is that correct?

Reply:
Correct.
Latest LDM of the year says "Make var have a nullable annotated type and infer the flow type as normal."

https://github.com/dotnet/csharplang/blob/master/meetings/2019/LDM-2019-12-18.md#var

Reply:
@CyrusNajmabadi @jcouv I see, I didn't know that, thank you for the info! In that case, would the appropriate solution be to table this fix until this change is implemented, and then take action from there? I am assuming that both the analyzer and the code fix provider will have to be modified to support the change.

Reply:
Yes, i think that would be fine.  However, i woudl also consider it high pri to do the "take action from there" step.  `var` preferences and `nrt` need to work well together as the language develops.  We should file an issue on this already to keep track of this.

tagging @dpoeschl for context so this doesn't get lost.

Reply:
Sounds good. 👍 I've opened up #40645 to track the change.